docs/markdown/Reference-manual.md

```diff
@@ -13,13 +13,20 @@ afterwards](#returned-objects).
   void add_global_arguments(arg1, arg2, ...)
 ```
 
-Adds the positional arguments to the compiler command line for the
-language specified in `language` keyword argument. If a list of
-languages is given, the arguments are added to each of the
-corresponding compiler command lines. Note that there is no way to
-remove an argument set in this way. If you have an argument that is
-only used in a subset of targets, you have to specify it in per-target
-flags.
+Adds the positional arguments to the compiler command line. This
+function has two keyword arguments:
+
+- `language` specifies the language(s) that the arguments should be
+applied to. If a list of languages is given, the arguments are added
+to each of the corresponding compiler command lines. Note that there
+is no way to remove an argument set in this way. If you have an
+argument that is only used in a subset of targets, you have to specify
+it in per-target flags.
+
+- `native` is a boolean specifying whether the arguments should be
+  applied to the native or cross compilation. If omitted, the flags
+  are added to native targets if compiling natively and cross targets
+  if cross compiling. Available since 0.48.0
```

Comment:
This para is a bit confusing. I would reword it as: 

```markdown
- `native` when set to `true`, the arguments will be added only for 
   native compilations when cross-compiling. If omitted or `false`, 
   the arguments are added to native targets if compiling natively
   and cross targets if cross compiling. Available since 0.48.0
```

Reply:
That's not the same wording. If set to `false` then they _won't_ be added to native targets ever. The table goes (or should go at least) like this:

```
true => only to native builds
false => only to cross builds
omitted => native builds when native compiling, cross builds _only_ when cross building
```

Reply:
Perhaps it's better to put it in a table form? Should be easier to parse than a paragraph.